apps/jubmoji-quest/src/pages/api/quests.ts

```diff
@@ -0,0 +1,117 @@
+import type { NextApiRequest, NextApiResponse } from "next";
+import prisma from "@/lib/prisma";
+import { getRandomNullifierRandomness } from "jubmoji-api";
+import { bigIntToHex } from "babyjubjub-ecdsa";
+
+export default async function handler(
+  req: NextApiRequest,
+  res: NextApiResponse
+) {
+  if (req.method === "GET") {
```

Comment:
I guess one actually somewhat clean feature of Next.js app router is the GET() and POST() api functions those are kinda clean